src/video_core/renderer_opengl/gl_state.h

```diff
@@ -144,6 +144,8 @@ class OpenGLState {
 
     std::array<bool, 2> clip_distance; // GL_CLIP_DISTANCE
 
+    GLuint renderbuffer;
```

Comment:
```suggestion
    GLuint renderbuffer; // GL_RENDERBUFFER_BINDING
```
(Explanation of this kind of comment in this file: assuming we don't have gl_state but still want to achieve the same effect of reducing opengl call when the state isn't changed, we would use `glGet(xxx)` to check the current state and skip changing the state when the value matches. Of course this is dumb because `glGet` itself is super slow. The comment is the "xxx" if we were to call `glGet`)

Reply:
Also, this needs a corresponding `ResetRenderbuffer` function for resetting the state when the buffer is deleted.